File: Increase/app/src/main/java/kr/ncspartner/increase/HolActivity.java
package kr.ncspartner.increase;

import android.os.Bundle;
import android.util.Log;
import android.view.View;
import android.widget.Button;
import android.widget.EditText;

import androidx.appcompat.app.AppCompatActivity;

public class HolActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_hol);

        EditText mEtMine    = findViewById(R.id.mEtMine);
        EditText mEtCom     = findViewById(R.id.mEtCom);
        EditText mEtResult  = findViewById(R.id.mEtResult);

        Button mBtn = findViewById(R.id.mBtn);

        mBtn.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                String[] list = {"홀", "짝"};
                int com = (int)(Math.random() * 2);
                mEtCom.setText(list[com]);

                if(mEtMine.getText().toString().equals(mEtCom.getText().toString())){
                    mEtResult.setText("맞았음");
                }else{
                    mEtResult.setText("틀렸음");
                }
            }
        });
    }
}

File: Hellokt/app/src/main/java/kr/ncspartner/hellokt/HolActivity.kt
package kr.ncspartner.hellokt

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button
import android.widget.EditText
import android.widget.TextView
import java.util.Random

class HolActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_hol)

        val mEtMine : EditText = findViewById(R.id.mEtMine)
        val mEtCom : EditText = findViewById(R.id.mEtCom)
        val mEtResult : EditText = findViewById(R.id.mEtResult)

        val mBtn : Button = findViewById(R.id.mBtn)
        val random = Random()

        mBtn.setOnClickListener{
                var a = arrayOf("짝", "홀")
                var com : Int = random.nextInt(2)

                mEtCom.setText(a[com])
                var mine : String = mEtMine.text.toString()

                if(mine.equals(a[com])){
                    mEtResult.setText("맞췄음")
                }else{
                    mEtResult.setText("틀렸음")
                }

        }
    }

}